tests/Fixer/Basic/BracesFixerTest.php

```diff
@@ -3482,10 +3482,10 @@ public function provideFixSpaceAroundTokenCases()
      * @param null|string $input
      * @param null|array  $configuration
      *
-     * @dataProvider provide55Cases
+     * @dataProvider provideFix55Cases
      * @requires PHP 5.5
      */
-    public function test55($expected, $input = null, array $configuration = null)
```

Comment:
Are you ok with renaming a few of the test methods along the way, or would you prefer not to?